build/feedparser.js

```diff
@@ -14,9 +14,10 @@ async function getFeedEntries(url) {
   for (const item of feed.rss.channel.item) {
     const description = cheerio.load(item.description);
     const summary = description("p").text();
+    const title = cheerio.load(item.title).text();
```

Comment:
I was thinking of writing some little `unescapeHTML` function or something but it feels better to rely on a trusted and mature library that I know I can trust.